corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -388,4 +392,107 @@ public void onError(Throwable throwable) {
 
         return streamDumper.getTxnRead().get();
     }
+
+    /**
+     * List all stream tags present in the Registry.
+     *
+     * @return stream tags
+     * */
+    public Set<String> listStreamTags() {
+        Set<String> streamTags = new HashSet<>();
+        runtime.getTableRegistry().getRegistryTable().values().forEach(record ->
+                streamTags.addAll(record.getMetadata().getTableOptions().getStreamTagList()));
+
+        log.info("\n======================\n");
+        log.info("Total unique stream tags: [{}]", streamTags.size());
+        streamTags.forEach(tag -> log.info(tag));
+        log.info("\n======================\n");
+
+        return streamTags;
+    }
+
+    /**
+     * List a map of stream tags to table names.
+     *
+     * @return map of tags to table names in the registry
+     */
+    public Map<String, List<TableName>> listTagsToTableMap() {
+        Map<String, List<TableName>> streamTagToTableNames = getMapTagToTableNames();
+        log.info("\n======================\n");
+        log.info("Total unique stream tags: [{}]", streamTagToTableNames.keySet().size());
+        log.info("Stream tags: {}\n", streamTagToTableNames.keySet());
+        streamTagToTableNames.forEach((tag, tableNames) -> printStreamTagMap(tag, tableNames));
+        log.info("\n======================\n");
+        return streamTagToTableNames;
```

Comment:
do we need to have a return value for all operations?  I saw the existing methods and they all return something but does not look like they need to...

Reply:
Returning the values will allow us to test better, this way unit tests can verify if the parsing is the exact expected output.